swim/memberlist.go

```diff
@@ -336,11 +336,12 @@ func (m *memberlist) Update(changes []Change) (applied []Change) {
 		// if change is local override, reassert member is alive
 		if member.localOverride(m.node.Address(), change) {
 			m.node.emit(RefuteUpdateEvent{})
+			newIncNo := nowInMillis(m.node.clock)
 			overrideChange := Change{
```

Comment:
@Motiejus The concern I have with copying the old change over constructing a new one is that this issue was actually cased by that exact case. In the nodejs version it used to extend the change it received, causing the source to not be overwritten and thus causing the change to take over the source address and incarnation number of the original change instead of putting its own information on there. I'd rather create a new change here as currently is done and run into more noticeable issues when we miss information from a change in here then have information fall through that should not have been copied.

If we want to go with the route of basing the change on the original change that came in we can do this with value types as well. If you assign a value type to a local variable it will copy all of its contents to the new struct and then allows us to update the fields we care about.